services/core-api/app/api/now_applications/resources/now_application_status_resource.py

```diff
@@ -141,6 +143,17 @@ def put(self, application_guid):
 
                 permit_amendment.save()
 
+                #Issue Permit as Verifiable Credential to OrgBook
+                try:
```

Comment:
Should any possible exceptions be blocking? Do we just want to log the messages?